signing_service/signing_service/tests/tests.py

```diff
@@ -13,13 +21,17 @@
 from signing_service.utils import is_valid_public_key
 
 
+concent_ecc_keys = ECCx(None)
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = concent_ecc_keys.raw_privkey, concent_ecc_keys.raw_pubkey
+
+
 class SigningServiceMainTestCase(TestCase):
 
     def setUp(self):
         self.host = '127.0.0.1'
         self.port = 8000
         self.initial_reconnect_delay = 2
-        self.concent_public_key = b'\xf3\x97\x19\xcdX\xda\x86tiP\x1c&\xd39M\x9e\xa4\xddb\x89\xb5,]O\xd5cR\x84\xb85\xed\xc9\xa17e,\xb2s\xeb\n1\xcaN.l\xba\xc3\xb7\xc2\xba\xff\xabN\xde\xb3\x0b\xa6l\xbf6o\x81\xe0;'
+        self.concent_public_key = CONCENT_PUBLIC_KEY
```

Comment:
Please don't hard-code keys. I see that this was already hard-coded in one of the previous issues so we can't really remove it though. Next time, if there's anything preventing you from not hard-coding it, please tell me. I think this was hard-coded because I forgot to add the command-line parameter for it in issue description.

Reply:
It is not in master yet but in pull request which I wanted to merge and on which this one is based, so I will replace it there.